pkg/project/project.go

```diff
@@ -150,7 +150,7 @@ func (p *Project) CommonTearDownSteps() error {
 	return nil
 }
 
-func (p *Project) Test() error {
+func (p *Project) Test(ctx context.Context) error {
```

Comment:
It would make sense to do these changes independently to have nicer PRs and less to confuse. We could just have had one PR upfront to get the interfaces straight and then fix the other things.

Reply:
Yes. You know how it is. You want to fix sth then everything else attacks you. I'll try to create other PR.

Reply:
Done.